playwright/.auth/user.json

```diff
@@ -0,0 +1,45 @@
+{
```

Comment:
is this not supposed to be in .gitignore?

Reply:
added to the gitignore